### PR TITLE
2457 - Link enabled in reference to data source column

### LINKED ISSUE #2457: National Data->Data sources comments: enable links in reference to data source column
Image

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #2457

https://user-images.githubusercontent.com/10047945/230638600-7a98921b-b393-4a3a-bf3d-56c5a271c1bb.mov

### CODE REVIEW

**Comment by sorja** [comment]
Looks good!

Should we add external link icon?

https://fontawesome.com/v5/icons/external-link-alt?f=classic&s=solid

**Review by sorja — APPROVED** [review]

**Comment by sorja** [comment]
I would maybe add placeholder for link "Link..."

**Comment by minotogna** [comment]
> Looks good!
> 
> 
> 
> Should we add external link icon?
> 
> 
> 
> https://fontawesome.com/v5/icons/external-link-alt?f=classic&s=solid

@sorja @rfontanarosa i think we should go through all the icons in a later stage . I would like to choose one and only one consistent icon set . What do you think ?

File: src/client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSources.scss [hunk 1]
```diff
@@ -5,16 +5,26 @@
 }
 
 .data-source-grid {
-  grid-template-columns: 1fr 1.3fr repeat(3, 1fr) 0.1fr;
+  grid-template-columns: 0.2fr 1fr 1fr repeat(3, 1fr) 0.1fr;
```
Comment (minotogna) [inline src/client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSources.scss:8]: The delete button width should be fixed in px. Is the 0.2fr the delete button width ?

File: src/client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSources.scss [hunk 2]
```diff
@@ -42,4 +48,19 @@
...
   }
+
+  .jodit-container {
+    border-top: none;
+    border-bottom: none;
+    border-right: none;
+  }
+
+  .data-source__markdown-wrapper {
+    padding: 10px;
```
Comment (minotogna) [inline src/client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSources.scss:59]: How does it look 8px with spacing-... ?

File: web-resources/img/icons.svg [hunk 3]
```diff
@@ -1,4 +1,16 @@
 <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+    <symbol id="text_fields" viewBox="0 0 24 24">
```
Comment (minotogna) [inline web-resources/img/icons.svg:2]: How about just text ?

File: web-resources/img/icons.svg [hunk 4]
```diff
@@ -1,4 +1,16 @@
...
+        <title>text fields</title>
+        <g class="nc-icon-wrapper">
+            <path d="M2.5 4v3h5v12h3V7h5V4h-13zm19 5h-9v3h3v7h3v-7h3V9z" fill="currentColor" stroke="none"></path>
+        </g>
+    </symbol>
+    <symbol id="insert_link" viewBox="0 0 24 24">
```
Comment (minotogna) [inline web-resources/img/icons.svg:8]: How about just link ?

**Pull request merged**